### PR TITLE
added Linear Search in C++ and Python

### PR DESCRIPTION
**Pull request by UtkarshChaurasia**
Linear Search added and readme updated for both C++ and Python.
For issue #12

### CODE REVIEW

File: C++/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ _add list here_
 4.Queue.
 5.Graph
 ## Searching
-_add list here_
+* [Linear Search](https://github.com/UtkarshChaurasia/NeoAlgo/blob/master/C%2B%2B/Linear_Search.cpp)
```
Comment (mar-galho) [inline C++/README.md]: Could you use the following format instead? Your link format points to your fork, and not to this repo :) Thanks!

`* [Linear Search](Linear_Search.cpp)`

File: Python/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
 _add list here_
 
 ## Searching
-_add list here_
+* [Linear Search](https://github.com/UtkarshChaurasia/NeoAlgo/blob/master/Python/Linear_Search.py)
```
Comment (mar-galho) [inline Python/README.md]: Could you use the following format instead? Your link format points to your fork, and not to this repo :) Thanks!

`* [Linear Search](Linear_Search.py)`

File: C++/README.md (hunk 1 above)
Comment (UtkarshChaurasia) [reply C++/README.md]: Thanks for reviewing. I'll make the changes!

File: Python/README.md (hunk 2 above)
Comment (UtkarshChaurasia) [reply Python/README.md]: Thanks for reviewing. I'll make the changes!

**Pull request closed without merging**